helm-chart-sources/pulsar/templates/autorecovery/autorecovery-deployment.yaml

```diff
@@ -178,7 +178,7 @@ spec:
           - name: ClusterName
             value: "{{ template "pulsar.fullname" . }}"
           - name: SuperRoles
-            value: {{ .Values.superUserRoles }}
+            value: "{{ .Values.superUserRoles }},{{ .Values.proxyRoles }}"
```

Comment:
What does this `SuperRoles` env var do here? I see it in other templates.

Reply:
That what Burnell calls the config option for the roles. https://github.com/datastax/burnell/blob/5a7c261e498ff5b34356b0c164d357e9f3a8b81b/src/workflow/keys-jwt.go#L98